edx_sg_block/sga.py

```diff
@@ -528,7 +530,7 @@ def enter_grade(self, request, suffix=''):
 
         state = json.loads(module.state)
         try:
-            score = int(score)
+            score = float(score)
```

Comment:
Why this change?

Reply:
since the database field is float and gradebook is working with float values